app/models/name/spelling.rb

```diff
@@ -60,7 +60,6 @@ def self.suggest_alternate_spellings(str)
     results
   end
 
-  private
```

Comment:
Why does RuboCop say the modifiers in this commit are redundant?
Is is because all subsequent methods are defined on self?